config/routes.rb

```diff
@@ -1,8 +1,10 @@
 Rails.application.routes.draw do
-  resources :families
-  resources :schools, only: [:index, :new, :create, :edit, :update, :destroy]
-  resources :trips, only: [:index, :new, :create, :edit, :update, :destroy]
-  resources :pages, only: [:show]
+  scope "(:locale)", locale: /en|it/ do
```

Comment:
What will happen if a user enters a locale that is not supported?

Reply:
@elle I thought that it would use the default one but now I checked...I couldn't find any solutions. I tried to modify the set_locale method but without success. I can set the locale to the default one but the path is still unknown...I'm sure there is an easy and straightforward way... 🙄

Reply:
I will implement it in another PR. I will merge this for now. Thanks for the tip!